docs/components/bottom_app_bars.md

```diff
@@ -0,0 +1,113 @@
+<!--docs:
+title: "Material bottom app bar"
+layout: detail
+section: components
+excerpt: "Bottom app bars provide access to a bottom navigation drawer and up to four actions, including the floating action button.
+iconId:
+path: /catalog/material-bottom-app-bar/
+-->
+
+# Bottom app bars
+
+[Bottom app bars](https://material.io/components/app-bars-bottom/) provide access to a bottom navigation drawer and up to four actions, including the floating action button.
+
+![Example bottom app bar with inset floating action button](assets/bottom_app_bars/bottom-app-bar-hero.png)
+
+## Using the bottom app bar
+
+**Note to developers: please add a link to your platform's FAB article below**
+
+Bottom app bars provide access to a bottom navigation drawer and up to four actions, including the [floating action button](link to floating action button here).
+
+Before you can use Material buttons, you need to import the Material Components package for Flutter:
+```
+package:flutter/material.dart
+```
+
+You need to use [`MaterialApp`](https://api.flutter.dev/flutter/material/MaterialApp-class.html).
+
+For more information on getting started with the Material for Flutter, go to the Flutter [Material library](https://api.flutter.dev/flutter/material/material-library.html) page.
+
+
+For more guidance using the bottom app bar, go to the [Material design page](https://material.io/components/app-bars-bottom/).
+
+## Bottom app bar example
+
+`BottomAppBar`
+* [Class description](https://api.flutter.dev/flutter/material/BottomAppBar-class.html)
+* [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/bottom_app_bar.dart)
+* [Dartpad demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/bottom_app_bars/regular)
+
+![Bottom app bar with with leading navigation drawer icon, overlapping FAB with '+' icon, and trailing 'search' and overflow menu control](assets/bottom_app_bars/bottom_app_bar_example.png)
```

Comment:
Might want to use the material colors here, like in the bottom navigation: https://github.com/material-components/material-components-flutter/blob/54202c067e9ac07a4b1f20d18b862b3e525a3fc7/docs/components/bottom_navigation.md


 backgroundColor: Color(0xFF6200EE),`